### ISSUE TITLE
Using Druid Kubernetes extension instead of Zookeeper ?

### ISSUE THREAD

**Issue by rdegez** [druid-io/druid-operator#173]
Hi,

Are you planning to use the Druid Kubernetes extensions in place of Zookeeper ?(https://druid.apache.org/docs/latest/development/extensions-core/kubernetes.html) 

Would reduce the number of dependencies :-)

**Comment by AdheipSingh** [comment]
Haven't used this but will be curious to see how does this behave :) 

@himanshug is the author of this extension, might be able to give some suggestions here.

**Comment by himanshug** [comment]
yeah, that makes sense. I will try and add one spec for that soon.

**Comment by AdheipSingh** [comment]
@himanshug i did some playaround with the extension. Here's a tiny yaml i tested with 
https://gist.github.com/AdheipSingh/43f6dc5b90d7e730deba604674d455b6 
can you validate this ? can add this in examples

**Comment by rdegez** [comment]
Hi @himanshug, happu new year! ;-)

Any chance to have a look at @AdheipSingh works ?
Removing the dependency on zookeeper seems to be a quick-win thanks to your extension andwould greatly improve cluster maintenance!

Cheers,

**Comment by AdheipSingh** [comment]
@rdegez https://github.com/druid-io/druid-operator/pull/241
you can take a look at this example, few cleanups needed from my end.
